### PR TITLE
From Poland with ❤️

### PR DESCRIPTION
**Pull request by Sija**
Several enhancements I've noticed goin' through the codebase.
I hope you don't mind I've used your plugin list as a base for [Wafalyzer](https://github.com/NeuraLegion/wafalyzer) 😅

Thanks for this tool!
Best

### CODE REVIEW

File: content/plugins/varnish.py [hunk 1]
```diff
@@ -11,9 +11,7 @@ def detect(content, **kwargs):
     headers = kwargs.get("headers", {})
     detection_schema = (
         re.compile(r"\bxid. \d+", re.I),
-        re.compile(r"varnish\Z", re.I),
-        re.compile(r"varnish"), re.I,
```
Comment (Sija) [inline content/plugins/varnish.py:15]: That looks like a typo btw 😅

**Comment by Ekultek** [comment]
https://github.com/Ekultek/WhatWaf/pull/925/files#diff-d2661472150f8c5b730119265b73e526L15 These two are the only problem. Also use the code however you want it’s free <3

**Comment by Sija** [comment]
> https://github.com/Ekultek/WhatWaf/pull/925/files#diff-d2661472150f8c5b730119265b73e526L15 These two are the only problem.

@Ekultek I'm not sure I understand, do you want me to close this PR or...?

> Also use the code however you want it’s free <3

Thanks ❤️

**Comment by Ekultek** [comment]
I don’t remember what I was going to say so I’m gonna merge this

**Pull request merged**